coreneuron/CMakeLists.txt

```diff
@@ -284,6 +284,15 @@ target_include_directories(coreneuron SYSTEM
 target_include_directories(coreneuron SYSTEM
                            PRIVATE ${CORENEURON_PROJECT_SOURCE_DIR}/external/CLI11/include)
 
+if(CORENRN_ENABLE_GPU)
+  # nrnran123.cpp possibly-temporarily uses Boost.Pool in GPU builds if it's available.
```

Comment:
Maybe we should create a follow-up ticket for replacing Boost with something lighter here?

Reply:
https://github.com/BlueBrain/CoreNeuron/issues/728